server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -30,6 +30,16 @@ const camelCaseToSlug = ( name ) => {
 		.replace( /^-/, '' );
 };
 
+const slugToCamelCase = name => {
+	if ( ! name ) {
+		return name;
+	}
+
+	return name
```

Comment:
I almost forgot, there's no `import` in vanilla nodejs, so to use the utility function from the client, it either has to be copy/pasted or the utility module has to have 0 dependencies (or use `require`).